modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA command line interface (CLI).
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+* Create a machine pool with a custom tag by running the following command:
++
+--
+[source,terminal]
+----
+$ rosa create machinepools --cluster=<name> --replicas=<replica_count> \
+     --tags=<key>=<value>,<effect>,<key>=<value>,<effect> \ <1>
+----
+<1> Replace `<key>=<value>,<effect>,<key>=<value>,<effect>` with a key, value, and effect for each tag.
+--
++
+.Example output
+[source,terminal]
+----
+$ rosa create machinepools --cluster=mycluster --replicas 2 --tags=key1:NoSchedule,key2:NoExecute --name=mp-1
```

Comment:
```suggestion
$ rosa create machinepools --cluster=mycluster --replicas 2 --tags='key1 value1,key2 value2' --name=mp-1
```

Reply:
NoSchedule is keyword for taint. Let's avoid causing confusion from customers

Reply:
Good idea - I tried to make it generic with this:

![image](https://github.com/openshift/openshift-docs/assets/16167833/a994a63b-91fc-4e44-9d5f-886c6f9cdb6a)